acceptance/helpers.rb

```diff
@@ -177,27 +204,29 @@ def host_to_info_s(host)
         "#{host} (#{host['platform']})"
       end
 
-      # Install puppet-agent on all agent nodes and connects them to the master.
-      # This is intended to set-up SUTs for tests that upgrade puppet-agent
-      # from some initial version. A key post-condition of this method is that
-      # all of the agents are ready to be upgraded with the puppet_agent module.
+
+      # Installs an initial puppet agent package on a host and connect the new agent node to the
+      # puppet master. This 'initial' agent is likely a lower version than the puppet master on
+      # purpose to facilitate an upgrade scenario.
       #
+      # @param [Beaker::Host] host The host
       # @param [String] initial_package_version_or_collection Either a version
       #   of puppet-agent or the name of a puppet collection to install the agent from.
-      # @yield Invokes any additional setup that's required. 
-      def set_up_agents_to_upgrade(initial_package_version_or_collection)
+      def set_up_initial_agent(host, initial_package_version_or_collection)
```

Comment:
I think this method should have `_on` appended, too, to make it clear that it expects a host